worldedit/manipulations.lua

```diff
@@ -217,8 +217,6 @@ function worldedit.copy2(pos1, pos2, off, meta_backwards)
 	dst_manip:set_param2_data(dst_data)
 
 	mh.finish(dst_manip)
-	src_data = nil
-	dst_data = nil

```

Comment:
This was intentional.

Reply:
Checking `worldedit\manipulations.lua`              2 warnings

    worldedit\manipulations.lua:220:2: value assigned to variable 'src_data' is unused
    worldedit\manipulations.lua:221:2: value assigned to variable 'dst_data' is unused

This can also be fixed by adding them to the `ignore` list.